### ISSUE TITLE
Bugs: Boat_with_three_swimmers

### ISSUE THREAD

**Issue by BenjaminRueth** [ChairOfStructuralMechanicsTUM/Mechanics_Apps#3]
I just observed some small bugs: 

- [ ] **Reset** button does not work, if pressed while animation is paused with **Pause** button. 
- [ ] **Reset** button while **Play** resets the animation, but if we choose i.e. 5 people and press **Reset**, then again **Play**, we only get a boat with 1 person even though the slider is still set to 5. I think there should be 5 people again.

What is the intended behaviour? Can we fix it?

**Comment by moustafabachar** [comment]
Yes sure, I will work on fixing that tomorrow. Thanks for the revision

**Comment by axelgreim** [comment]
Hey @moustafabachar, 
Thanks for your contribution the Boat and Swimmer App. I Like the velocity diagram. But there are still a couble of improvements left. I've updated the Redme.md of this app. Could you please solve the To-Dos? afterwards we can close this issue...

**Comment by axelgreim** [comment]
I've finalized the Boat with three swimmers. It's already online. 
- no Bugs left, Colors according to styleguide of TUM...
